src/components/LeftSidebar/ConversationsList/Conversation.vue

```diff
@@ -45,17 +45,14 @@
 				{{ conversationInformation }}
 			</template>
 		</template>
-		<template v-if="!isSearchResult" slot="actions">
+		<template v-if="!isSearchResult" #actions>
 			<NcActionButton v-if="canFavorite"
 				:close-after-click="true"
 				@click.prevent.exact="toggleFavoriteConversation">
-				<Star v-if="item.isFavorite"
-					slot="icon"
-					:size="20" />
-				<Star v-else
-					slot="icon"
-					:size="20"
-					:fill-color="'#FFCC00'" />
+				<template #icon>
+					<Star v-if="item.isFavorite" :size="20" />
+					<Star v-else :size="20" :fill-color="'#FFCC00'" />
```

Comment:
```suggestion
					<Star v-else :size="20" fill-color="#FFCC00" />
```

😊

Reply:
I wanted to have this commit plain, having only one lint rule fix, related to the migration.

For this thing we can add a new rule with the auto fix.